compiler/test/dotty/tools/vulpix/ParallelTesting.scala

```diff
@@ -55,14 +55,14 @@ trait ParallelTesting extends RunnerOrchestration { self =>
 
     def runClassPath: String = outDir.getAbsolutePath + ":" + flags.runClassPath
 
-    def title: String = self match {
+    def title: String = (self match {
       case self: JointCompilationSource =>
         if (self.files.length > 1) name
         else self.files.head.getPath
 
       case self: SeparateCompilationSource =>
         self.dir.getPath
-    }
+    }).stripPrefix("../")
```

Comment:
What about we change the working directory when running tests? Something like:
```scala
baseDirectory in test := baseDirectory in ThisBuild
```
We could get rid of all the `"../` in `CompilationTest`